### DISCUSSION TITLE
The url previews for Tweakers.net never work

### DISCUSSION THREAD

**Discussion by thibaultmol** [mastodon/mastodon#32853] [Q&A]
https://mastodon.social/@Tweakers/113470129396562062

it always shows the cookie/privacy page instead of the actual article
I assume there's an issue about problems like this but i can't find which one is relevant for something like this

**Comment by mjankowski** [comment]
The software doesnt really understand whatever its getting back. If the thing that comes back from a URL is a privacy/warning/cookie/whatever page ... its going to think that's the content, and its going to make a preview card about that thing.

**Reply by thibaultmol** [reply]
shouldn't it just opengraph data?
I thought that was the whole point of opengraph, so that the website in question can easily tell an embedder what the page is about:

https://www.opengraph.xyz/url/https%3A%2F%2Ftweakers.net%2Freviews%2F12658%2Fnuki-smart-lock-ultra-kleiner-sneller-krachtiger.html

<img width="1411" height="853" alt="image" src="https://github.com/user-attachments/assets/87d1056d-d3f6-40c3-a37f-64885a6cb81c" />

**Reply by mjankowski** [reply]
Sorry I hadnt fully explored the page.

Yes, it should do that ... and if that data is there my guess is more of an intermittent issue than something permanent.

For example, it looks like more recent posts in that account do show the preview correctly?
